### DISCUSSION TITLE
Help to install the visual workflow UI

### DISCUSSION THREAD

**Discussion by alawth** [Netflix/conductor#2615] [Q&A]
I have tried to start the conductor server from source (https://github.com/Netflix/conductor) and executed the following.

./gradlew server

/ui/ -> yarn run start

And I'm getting this UI [here](https://ibb.co/yhMf1fv) and not the workflow visual UI that I'm expecting to have [here](https://user-images.githubusercontent.com/3489862/55858537-237d0d80-5b9a-11e9-9e62-0d98d1b5c780.png) . 

Any idea on how can I install or enable the visual workflow UI?

Thanks.

**Comment by andyyumiao** [comment]
@alawth the workflow visual UI that you mentioned is just old conductor version, so you can downgrade conductor to version 2.31.

**Reply by alawth** [reply]
Thanks @andyyumiao. With the latest version what is the recommended UI tool for workflow visualisation? since the [default UI](https://ibb.co/yhMf1fv) only list the workflows in tabular format. Thanks.

**Comment by alawth** [comment]
Any general recommendations to have workflow visual UI with the latest conductor version (> 3) since it is not available on the default [UI](https://ibb.co/yhMf1fv) tool ? Thanks.

**Comment by peterlau** [comment]
The v3.x UI should visualize workflows as a directed graph in the same way as the 2.x UI. Can you elaborate on the difficulty you are encountering?

**Reply by laxmikantb** [reply]
I downloaded latest conductor, I see the workflow definitions directed graph, but not able to see the execution directed graph, Is there any step I am missing, I see the following queries for workflow UI but no output in workflow executions ui.


["default","/workflow/search?",{"freeText":"*","page":1,"query":"","rowsPerPage":15,"sort":"startTime:DESC"}]
1
["default","/workflow/search?",{"freeText":"*","page":1,"query":"workflowType IN (date-to-year-rest-post-dateformat-2)","rowsPerPage":15,"sort":"startTime:DESC"}]
1
["default","/metadata/workflow"]

**Comment by autodidactic** [comment]
i have tried to use localhost:8080 as swagger UI for posting task and workflow def and also started workflow but the UI on localhost:5000 is not able to show the visual status of execution.


seems like there a disconnect between visual UI and workflow execution while using in memory persistence
